File: java/Human.java
public class Human{

    String nama;
    String nik;
    String gender;


 //Konstruktor
    public Human(){
        //konstruktor kosong
    }

    public Human(String nama, String nik, String gender){
        //konstruktor dengan parameter
        this.nama = nama;
        this.nik = nik;
        this.gender = gender;
    }

    //setter getter

    //SETTER
    public void setnama(String nama){
        this.nama = nama;
    }

    public void setnik(String nik){
        this.nik = nik;
    }

    public void setgender(String gender){
        this.gender = gender;
    }

    //GETTER
    public String getnama(){
        return this.nama;
    }

     public String getnik(){
        return this.nik;
    }

    public String getgender(){
        return this.gender;
    }

}

File: python/human.py
class Human:

    def __init__(self, NIK = "", nama = "", gender = ""):
        self.__NIK = NIK
        self.__nama = nama
        self.__gender = gender

    def getNIK(self):
        return self.__NIK

    def setNIK(self, NIK):
        self.__NIK = NIK

    def getnama(self):
        return self.__nama

    def setnama(self, nama):
        self.__nama = nama

    def getgender(self):
        return self.__gender

    def setgender(self, gender):
        self.__gender = gender